extensions/binding/org.eclipse.smarthome.binding.dmx/src/main/java/org/eclipse/smarthome/binding/dmx/internal/DmxBridgeHandler.java

```diff
@@ -164,7 +164,7 @@ protected void installScheduler() {
             uninstallScheduler();
         }
         if (refreshTime > 0) {
-            senderJob = scheduler.scheduleAtFixedRate(packetSender, 1, refreshTime, TimeUnit.MILLISECONDS);
+            senderJob = scheduler.scheduleWithFixedDelay(packetSender, 1, refreshTime, TimeUnit.MILLISECONDS);
```

Comment:
The DMX protocol is pretty sensitive wrt timing - my guess is that here the `...AtFixedRate` is by intention. 
@J-N-K could you please have a quick look?

Reply:
It should stay with `scheduleAtFixedRate`. It was chosen exactly because of the timing require,emnts of the DMX protocol. Shall I add a comment for that?

Sorry, didn't see this earlier.